x-pack/test/security_solution_cypress/ai4dsoc_serverless_config.ts

```diff
@@ -34,6 +34,7 @@ export default async function ({ readConfigFile }: FtrConfigProviderContext) {
         ])}`,
         '--csp.strict=false',
         '--csp.warnLegacyBrowsers=false',
+        '--xpack.features.overrides.siemV2.hidden=true',
         '--xpack.features.overrides.securitySolutionTimeline.hidden=true',
         '--xpack.features.overrides.securitySolutionNotes.hidden=true',
```

Comment:
Also, I still don't get why we need to duplicate these overrides:
```ts
        '--xpack.features.overrides.securitySolutionTimeline.hidden=true',
        '--xpack.features.overrides.securitySolutionNotes.hidden=true',
```
The test server should already have that setup from the tier config (`config/serverless.security.search_ai_lake.yml`)